my-app/src/components/login/LoginForm.js

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import InputDetail from "../shared/Input";
+import RegButton from "../shared/Button";
```

Comment:
It should be just a Button, all the components in the shared folder are common and shouldn't have specific names. The same for InputDetails